### PR TITLE
sync ci profiles across versions - part 6 - aws-ipi-disc-priv

### PR DESCRIPTION
**Pull request by jianlinliu**
All stable/upgrade ci profiles are selected from nightly ci as source, so sync all the aws-ipi-disc-priv ci profiles to align with the source nightly ci.

### CODE REVIEW

**Comment by jianlinliu** [comment]
cc @JianLi-RH and @shellyyang1989 to review.

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly-4.14-upgrade-from-stable-4.13.yaml [hunk 1]
```diff
@@ -86,25 +86,15 @@ tests:
...
-    workflow: cucushift-installer-rehearse-aws-ipi-disconnected-private
-- as: aws-ipi-disc-priv-sno-f28
+- as: aws-ipi-disc-priv-sno-basecap-none-f28
   cron: 3 8 9 * *
   steps:
     cluster_profile: aws-qe
     env:
       BASE_DOMAIN: qe.devcluster.openshift.com
+      BASELINE_CAPABILITY_SET: None
+      ENABLE_OTA_TEST: OCP-24663
```
Comment (JianLi-RH) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly-4.14-upgrade-from-stable-4.13.yaml:96]: Do you really want to enable the case?

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-stable-4.17-upgrade-from-stable-4.16.yaml [hunk 2]
```diff
@@ -79,16 +79,6 @@ tests:
     test:
     - chain: openshift-upgrade-qe-sanity-disconnected
     workflow: cucushift-installer-rehearse-aws-ipi-disconnected-private-edge-zone
-- as: aws-ipi-disc-priv-sno-f28
```
Comment (JianLi-RH) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-stable-4.17-upgrade-from-stable-4.16.yaml:82]: Is there any reason to remove this job?

Comment (jianlinliu) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-stable-4.17-upgrade-from-stable-4.16.yaml:82]: All the stable profiles are selected from nightly profiles pools, there is no `aws-ipi-disc-priv-sno` profile any more, so remove it here.

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly-4.14-upgrade-from-stable-4.13.yaml (hunk 1 above)
Comment (jianlinliu) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly-4.14-upgrade-from-stable-4.13.yaml:96]: Yes, `aws-ipi-disc-priv` is being replaced by `aws-ipi-disc-priv-sno-basecap-none`.

**Comment by JianLi-RH** [comment]
For the upgrade part, LGTM

**Comment by jianlinliu** [comment]
@gpei can you help review?

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__multi-nightly.yaml [hunk 3]
```diff
@@ -893,25 +893,28 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-azure-ipi-disk-encryption-set-minimal-permission
-- as: azure-ipi-disc-fullypriv-amd-mixarch-f28-destructive
-  cron: 35 4 28 * *
+- as: azure-ipi-disc-fullypriv-arm-mixarch-f2
```
Comment (gpei) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__multi-nightly.yaml:896]: @jianlinliu are these changes on `azure-ipi-disc` on purpose in this PR?

Comment (jianlinliu) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__multi-nightly.yaml:896]: ah, I just re-order the two jobs, make no other change.

Comment (gpei) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__multi-nightly.yaml:896]: ok, then let's just ignore it

**Comment by jianlinliu** [comment]
cc @liangxia to review.

**Pull request merged**